starter-code/css/main.css

```diff
@@ -0,0 +1,126 @@
+body {
+  border-left:5px #ccc solid;
+  margin:0 0 0 5px;
+  font-family: Lato;
+}
+
+header {
+  max-width:700px;
+  padding-left:24px;
+}
+
+article {
+  max-width:700px;
+  padding-left:24px;
+}
+
```

Comment:
Since you are using the same two properties and values here, you should combine the selectors:
```css
header, article, footer {
  max-width:700px;
  padding-left:24px;
}
```